### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by TimZib**

### CODE REVIEW

File: 2. Objects and Functions.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+let result = showWarehouse(apples, pears);
+
+alert( result );
+
+//
+
+let name = prompt("Введите Ваше имя");
+
+function terminal(name) {
+    if (name == "Tsimur") {
```
Comment (YauhenEp) [inline 2. Objects and Functions.js:27]: it's better to use === for comparison

File: 2. Objects and Functions.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+showType(8, "EPAM", true);
+
+//
+
+let num = prompt("Введите число");
+
+function isPrime(num) {
+    for (let i = 2; i < num; i++) {
+      if ( num % i == 0) return alert (false);
```
Comment (YauhenEp) [inline 2. Objects and Functions.js:52]: use strict comparison

File: 2. Objects and Functions.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+showType(8, "EPAM", true);
+
+//
+
+let num = prompt("Введите число");
+
+function isPrime(num) {
+    for (let i = 2; i < num; i++) {
```
Comment (YauhenEp) [inline 2. Objects and Functions.js:51]: what is the for loop here for, it does not play any role.

File: 2. Objects and Functions.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+showType(8, "EPAM", true);
+
+//
+
+let num = prompt("Введите число");
+
+function isPrime(num) {
+    let i = 2; i < num; i++;
```
Comment (YauhenEp) [inline 2. Objects and Functions.js:51]: it does not work, because this string is redundant

Comment (TimZib) [reply 2. Objects and Functions.js]: I do not find another way to check if a number is a prime number

**Pull request merged**